packages/amplify_authenticator/lib/src/services/amplify_auth_service.dart

```diff
@@ -38,6 +39,8 @@ abstract class AuthService {
 
   Future<AuthUser?> get currentUser;
 
+  Future<bool> get isValidSession;
```

Comment:
See comment below. Making this a getter is a little confusing.
```suggestion
  Future<bool> isValidSession();
```